lms/djangoapps/course_blocks/transformers/user_partitions.py

```diff
@@ -192,8 +158,13 @@ def collect(cls, block_structure):
     def transform(self, user_info, block_structure):
         """
         Mutates block_structure and block_data based on the given user_info.
+
+        Arguments:
+            user_info (object)
+            block_structure (BlockStructureCollectedData)
         """
-        # TODO 8874: Factor out functionality of UserPartitionTransformation.apply and access._has_group_access into a common utility function.
+        # TODO 8874: Factor out functionality of UserPartitionTransformation.transform and
+        # access._has_group_access into a common utility function.
```

Comment:
We should update this TODO.  Instead of what's written here, we should implement https://openedx.atlassian.net/browse/MA-1019.